CHANGES.rst

```diff
@@ -4,6 +4,10 @@
 New Features
 ------------
 
+- Ability to cycle through datasets to expose information during mouseover. [#1953]
```

Comment:
Does this new ability need to be documented over here?

* https://jdaviz.readthedocs.io/en/latest/imviz/displayimages.html#cursor-information
* https://jdaviz.readthedocs.io/en/latest/specviz/displaying.html#cursor-information

Reply:
I'm not sure, we're definitely planning to iterate on the design in the (hopefully near) future.  Do you have suggestions for something to add there?  Just briefly stating that you can cycle through the layers or request cursor-only information?

Reply:
Something brief is okay. Otherwise, it is basically undocumented as far as the user is concerned. Not everyone reads the change log.